libavfilter/vf_dnn_detect.c

```diff
@@ -203,10 +203,40 @@ static int read_detect_label_file(AVFilterContext *context)
     return 0;
 }
 
+static int check_output_nb(DnnDetectContext *ctx, DNNBackendType backend_type, int output_nb)
```

Comment:
the commit title is: lavfi/dnn_backend_tensorflow.c: support detect model

why vf_dnn_detect.c is changed in this patch.

Reply:
@Semmer2  just in case you missed the comment.  And please send the patch to community as we talked.

Reply:
The commit title has been changed to 'lavfi/dnn_backend_tensorflow'. And i sent them to community just now, thank you